include/config/item.h

```diff
@@ -13,6 +13,7 @@
 #define I_GEM_BOOST_POWER               GEN_LATEST  // In Gen5+, the Gem boost power was reduced from 50% to 30%.
 #define I_USE_EVO_HELD_ITEMS_FROM_BAG   FALSE       // If TRUE, items such as Razor Claw or Electirizer will be usable from the bag to evolve a Pokémon just like in LA.
 #define I_TYPE_BOOST_POWER              GEN_LATEST  // In Gen4+, all regular type boosting held items had their power increased from 10% to 20%. eg. Charcoal
+#define I_SELL_VALUE                    GEN_LATEST  // In Gen9+, items sell for 1/4 of their value instead of 1/2.
```

Comment:
```suggestion
#define I_SELL_VALUE_FRACTION           GEN_LATEST  // In Gen9+, items sell for 1/4 of their value instead of 1/2.
```

Reply:
I would add a separate config for updated prices instead of linking the valuable item prices to this config.

Reply:
Then I'd have to go through and see how prices changed over all games, which I don't really want to do in this pr.

Reply:
Then don't update the prices at all in this PR.

Reply:
Alright.